roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/preview/RoborazziPreviewTool.kt

```diff
@@ -160,6 +165,22 @@ class RoborazziPreviewPanel(project: Project) : JPanel(BorderLayout()) {
         anchor = GridBagConstraints.WEST
         insets = JBUI.insets(4)
       })
+      add((JBTextField().apply {
+        emptyText.text = "Enter screenshot name..."
```

Comment:
This is a placeholder(hint text) for a text field.
Please let me know if you have any requests as I have inserted a tentative description.